<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopSearchClass</class>
 <widget class="QMainWindow" name="DesktopSearchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DesktopSearch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="desktopsearch.qrc">
    <normaloff>:/DesktopSearch/Design/Plane.png</normaloff>:/DesktopSearch/Design/Plane.png</iconset>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget
{
	background-image: url(:/DesktopSearch/Design/mainBack.png);
	background-repeat:no-repeat;
}</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>45</y>
      <width>971</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit {
	background-image: url(:/DesktopSearch/Design/SearchBarBack.png);
	border:0px;
	background-repeat:no-repeat;
	background-color:transparent;
	padding:2px 26px 0px;
}
#lineEdit:focus {
    	background-image: url(:/DesktopSearch/Design/SearchBarBack.png);
}

#lineEdit:edit-focus {
    
}
</string>
    </property>
    <property name="placeholderText">
     <string>What you want here...</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>923</x>
      <y>48</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton
{
	background-image: url(:/DesktopSearch/Design/SearchBtn.png);
	background-repeat:no-repeat;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>111</y>
      <width>1004</width>
      <height>533</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#tableWidget
{
	background:transparent;
}
QHeaderView::section
{
	background:#e5e5e5;
	border: 0px;
	border-right: 1px solid grey;
	height:26px
}
QTableWidget::item
{
	text-align: left;
}
QTableWidget::item:selected {
	background-image: url(:/DesktopSearch/Resources/tableSelectRowBackground.png);
	background-repeat: repeat-x;
	background-position: top;
	border:0px dashed #bbb;
	background-color: #458bff;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DotLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>645</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>645</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>74</y>
      <width>964</width>
      <height>191</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#listView{
	background-color: rgba(250, 250, 250, 200);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="desktopsearch.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DesktopSearchClass</receiver>
   <slot>OnSearchBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>968</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>DesktopSearchClass</receiver>
   <slot>OnSearchBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DesktopSearchClass</receiver>
   <slot>OnEdit(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>DesktopSearchClass</receiver>
   <slot>OnItemDoubleClicked(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>DesktopSearchClass</receiver>
   <slot>OnHistoryItemDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DesktopSearchClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1017</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSearchBtnClicked()</slot>
  <slot>OnEdit(QString)</slot>
  <slot>OnItemDoubleClicked(int,int)</slot>
  <slot>OnHistoryItemDoubleClicked(QModelIndex)</slot>
 </slots>
</ui>
